### ISSUE TITLE
[CLI] dependencies crashing

### ISSUE THREAD

**Issue by kevinchern** [UBC-Stat-ML/blangSDK#138]
Current code not robust.
Reproduce by creating a dependencies.txt as follows:
```
ca.ubc.stat:conifer:2.0.4

```
**note empty line**
**NOT**
```
ca.ubc.stat:conifer:2.0.4
```
call `blang` cli and it should crash. 
standalone compiler tries to do `compile ''`

**Comment by alexandrebouchard** [comment]
Good point, and we might as well add a bit more validation that each line should have the format "^.+[:].+[:].+$" and print a meaningful message if it does not.

**Comment by kevinchern** [comment]
we will send a pull request soon :)!
